eng/Versions.props

```diff
@@ -6,6 +6,8 @@
     <UsingToolXliff>false</UsingToolXliff>
   </PropertyGroup>
   <PropertyGroup>
+    <SystemRuntimeCompilerServicesUnsafeVersion>6.0.0</SystemRuntimeCompilerServicesUnsafeVersion>
+    <SystemNumericsVectorsVersion>4.5.0</SystemNumericsVectorsVersion>
```

Comment:
These versions aren't updated by dependency flow as they aren't produced live anymore. Move them into Directory.Packages.props directly into the PackageVersion items.

Reply:
> These versions aren't updated by dependency flow

For my own education, can you please tell me how did you know this? Where did you look to confirm this info?

Reply:
I was part of the PR that stopped shipping the Unsafe package in runtime so I knew about that one but just in general, those are corefx / runtime packages so I would expect them to have a 8.0.0 version on nuget.org if they would still live build.